daniel-davidF/styles/base.css

```diff
@@ -0,0 +1,43 @@
+/* DONE: Add a rule for images, so they never exceed their container's width. */
+img{
+  overflow: hidden;
+  width: 20vw;
+  height: 20vh;
+  
```

Comment:
This warps your images unnecessarily. You should make use of `max-width` instead to ensure that the images do not get too large.